lib/matplotlib/scale.py

```diff
@@ -172,16 +179,12 @@ def forward(values: array-like) -> array-like
         self._transform = transform
 
     def get_transform(self):
-        """
-        The transform for arbitrary scaling
-        """
+        """Return the `.FuncTransform` associated with this scale."""
+        # docstring inherited
```

Comment:
well, right now it's not inherited :) but per the above I would suggest to inherit it